### PR TITLE
add JJB template to enable edgex-global-pipelines build

### PR DESCRIPTION
**Pull request by ernestojeda**
Signed-off-by: Ernesto Ojeda <ernesto.ojeda@intel.com>

### CODE REVIEW

**Review by lranjbar — APPROVED** [review]
👍

**Comment by soda480** [comment]
Lgtm

**Review by soda480 — APPROVED** [review]
LGTM

File: jjb/global-pipelines/edgex-global-pipelines.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+
+- project:
+    name: edgex-global-pipelines
+    project-name: edgex-global-pipelines
+    project: edgex-global-pipelines
+    mvn-settings: EdgeX
```
Comment (eb-oss) [inline jjb/global-pipelines/edgex-global-pipelines.yaml]: The "EdgeX" settings file is not a maven file. What are you trying to get it to provide?

**Review by eb-oss — CHANGES_REQUESTED** [review]
Has this been tested on the sandbox?

File: jjb/global-pipelines/edgex-global-pipelines.yaml (hunk 1 above)
Comment (ernestojeda) [reply jjb/global-pipelines/edgex-global-pipelines.yaml]: Yes I tested this on the sandbox...

Comment (ernestojeda) [reply jjb/global-pipelines/edgex-global-pipelines.yaml]: I have requested maven settings on service desk

Comment (ernestojeda) [reply jjb/global-pipelines/edgex-global-pipelines.yaml]: Sandbox job:
https://jenkins.edgexfoundry.org/sandbox/blue/organizations/jenkins/edgex-global-pipelines-verify-pipeline/detail/edgex-global-pipelines-verify-pipeline/1/pipeline

**Pull request merged**